.github/renovate.json

```diff
@@ -31,11 +32,25 @@
             "matchStrings": [
                 "BLDR_RELEASE\\s+\\?=\\s+(?<currentValue>.*)\\n"
             ],
-            "datasourceTemplate": "github-releases",
-            "depNameTemplate": "siderolabs/bldr"
+            "depNameTemplate": "siderolabs/bldr",
+            "datasourceTemplate": "github-tags",
+            "versioningTemplate": "semver"
         }
     ],
     "packageRules": [
+        {
+            "matchPackageNames": [
+                "golang/go",
+                "nvidia/open-gpu-kernel-modules"
+            ],
+            "versioning": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.?(?<patch>\\d+)?$"
+        },
+        {
+            "matchPackageNames": [
+                "https://sourceware.org/git/glibc.git"
+            ],
+            "versioning": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.?(?<patch>(?!9000$)\\d+)?$"
```

Comment:
glibc publishes tags with `.9000` suffix, but not actual tarball releases, so filter them out

Reply:
```bash
❯ git ls-remote https://sourceware.org/git/glibc.git | grep 9000
7394a7690b9d00e7d743bb997b18b73e5eb3e61e	refs/tags/glibc-2.26.9000
00cdcf5a4110f7ac68651f5662693c82f7bffaca	refs/tags/glibc-2.26.9000^{}
b531d7cb8cf04ece600c30cd1afaf5e011dfb399	refs/tags/glibc-2.27.9000
086ee48eaeaba871a2300daf85469671cc14c7e9	refs/tags/glibc-2.27.9000^{}
4a793237825b5bdce33229a9d562a5813f1b30ff	refs/tags/glibc-2.28.9000
437faa9675dd916ac7b239d4584b932a11fbb984	refs/tags/glibc-2.28.9000^{}
cbcc680c774793034aa122ae2fa9b52be29e4dfd	refs/tags/glibc-2.29.9000
de44ab67aa4eec369deea828733567c35a0611c0	refs/tags/glibc-2.29.9000^{}
34c2f244d6031c8a0656fa894d610d980a10a234	refs/tags/glibc-2.30.9000
0b8c2f95df589c9c1ad25312475049bcc7ed7059	refs/tags/glibc-2.30.9000^{}
7bc5b9f376038baabfddac70204d58163234985f	refs/tags/glibc-2.31.9000
d8775136b5830394d50c98997983b96a236629bf	refs/tags/glibc-2.31.9000^{}
6e76ab513ac8c35bea76d882d8ccb8636a33fa74	refs/tags/glibc-2.32.9000
4c56bcbceb05b44965d48e701711f850b83d7c69	refs/tags/glibc-2.32.9000^{}
f49ea61a97f7b9258dcba08e298933012b25f4d1	refs/tags/glibc-2.33.9000
2edf8f3d754407e04f534df18a2ebe04175cda48	refs/tags/glibc-2.33.9000^{}
a91d3da25430669ef5020ae6a7f9963ab528dcc8	refs/tags/glibc-2.34.9000
a85c93c42433aa52d4c5bc18e1dff038a82415bf	refs/tags/glibc-2.34.9000^{}
85641d24875e8b8a3a18370ec359442839a821c7	refs/tags/glibc-2.35.9000
bc6c1af537dbcc05e1559e02f0f12942a829c224	refs/tags/glibc-2.35.9000^{}
6e5aa88fe24014522bf307bbc9b686260b23008e	refs/tags/glibc-2.36.9000
5fded9c44537283593a7f839347f831e15726572	refs/tags/glibc-2.36.9000^{}
d8950c7e6bce731032137b190cfbace06b45133f	refs/tags/glibc-2.37.9000
2412deae1af0ca37b096ff73517486e7d1e9fe46	refs/tags/glibc-2.37.9000^{}
```